target/linux/lantiq/image/Makefile

```diff
@@ -757,89 +860,107 @@ endif
 ifeq ($(SUBTARGET),falcon)
 
 define Device/lantiq_easy98000-nor
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000 Falcon Eval Board
+  DEVICE_VARIANT := NOR
   IMAGE_SIZE := 3904k
   DEVICE_DTS := EASY98000NOR
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board NOR
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-nor
 
 define Device/lantiq_easy98000-nand
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000 Falcon Eval Board
+  DEVICE_VARIANT := NAND
   IMAGE_SIZE := 3904k
   DEVICE_DTS := EASY98000NAND
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board NAND
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-nand
 
 define Device/lantiq_easy98000-sflash
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000 Falcon Eval Board
+  DEVICE_VARIANT := SFLASH
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY98000SFLASH
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board SFLASH
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-sflash
 
 define Device/lantiq_falcon-mdu
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := MDU Lantiq Falcon / VINAXdp MDU Board
```

Comment:
"Falcon / VINAXdp MDU Board"?

Reply:
I've changed it.. but I cannot find any information. So this is a bit of a stretch.